apps/web/.env.example

```
@@ -1,7 +1,11 @@
 NEXT_PUBLIC_SANITY_PROJECT_ID=
-NEXT_PUBLIC_SANITY_DATASET=
+NEXT_PUBLIC_SANITY_DATASET=production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SANITY_DATASET key should go before the NEXT_PUBLIC_SANITY_PROJECT_ID key

(UnorderedKey)